StudyPathTraker/StudyPathTraker/Collection/SPCategoriesViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  SPCategoriesViewController.swift
+//  StudyPathTraker
+//
+//  Created by Rafael Lopez on 3/30/18.
+//  Copyright © 2018 Jerti. All rights reserved.
+//
+
+import UIKit
+
+class SPCategoriesViewController: UIViewController {
+    
+    @IBOutlet weak var flowLayout: SPCategoriesCollectionViewFlowDelegate! {
+        didSet {
+            flowLayout.selectedItemAction = { [weak self] index in
```

Comment:
this is implement twice one here and the other on `setUpCollectionView` method. Would be better if you remove this

Reply:
ready, removed